samcli/hook_packages/terraform/hooks/prepare/exceptions.py

```diff
@@ -54,7 +64,7 @@ def __init__(self, local_variable_reference, dest_resource_list):
             issue_link=LOCAL_VARIABLES_SUPPORT_ISSUE_LINK,
             line_sep=os.linesep,
         )
-        UserException.__init__(self, msg)
+        ApplyLimitationException.__init__(self, msg)
```

Comment:
just curious and non-blocking - why use `ApplyLimitationException.__init__(self, msg)` instead of `super().__init__(self, msg)`?

Reply:
No particular reason, the rest of the exceptions in this file reference the class instead of using `super()`, so instead of changing everything over to `super`, I just kept the existing behaviour. We could go either way since we are only inheriting from one parent class.